SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -740,6 +740,7 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
     AD::SetPreaccIn(Vorticity_i, 3);
     AD::SetPreaccIn(V_i[idx.Density()], V_i[idx.LaminarViscosity()], V_i[idx.EddyViscosity()]);
     AD::SetPreaccIn(V_i[idx.Velocity() + 1]);
+    AD::SetPreaccIn(V_i[idx.SoundSpeed()]);
```

Comment:
OMG. I'm fully forgot for incompressible solver in SU2. 

If the incompressible solver and compressibility options are selected together when reading the configuration file, it would be better to make the su2 code break with error messages. 

I'll update this and PR again.

Reply:
I checked the "V_i[idx.SoundSpeed()];" in incompressible RANS, the "V_i[idx.SoundSpeed()];" is the same with "V_i[idx.Beta()];" (I don't know beta means exactly).

See CIncEulerSolver.cpp and CIncEulerVariable.hpp.

Reply:
Beta is a pseudo speed of sound for low speed preconditioning, it's something like 2 times the maximum velocity in the domain.

Reply:
Thanks for the explanation. 

I added to config.cpp so that the code breaks when both INC_RANS and Compressibility Correction are selected.